### DISCUSSION TITLE
Support new parent field for company-managed projects

### DISCUSSION THREAD

**Discussion by andrebask** [ankitpokhrel/jira-cli#764] [Ideas]
Add support for the new parent field so that it is possible to create issues linked to an epic in company-managed projects.

Creating an issue linked to an Epic in company-manged projects is currently impossible. This is because of the [new Parent field](https://community.atlassian.com/t5/Jira-articles/Introducing-the-new-Parent-field-in-company-managed-projects/ba-p/2377758) introduced for these kind of projects. See also https://github.com/ankitpokhrel/jira-cli/issues/637

**Comment by ankitpokhrel** [comment]
Could you check if setting the project type to `next-gen` works? https://github.com/ankitpokhrel/jira-cli/issues/738#issuecomment-2203847105

**Reply by andrebask** [reply]
Yeah that works, thanks!
